content/en/docs/armory-agent/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent for K8s"
+description: >
+  A native Kubernetes service that enables high-scale deployments to Kubernetes using Spinnaker<sup>TM</sup>
```

Comment:
Do we really the ™? We don't put it anywhere else and it looks ... odd.

Reply:
Putting ™ on at least the first mention of Spinnaker requirements for using the Spinnaker trademark based on the discussion between the Linux Foundation lawyers and ours. We need to have it on every page for the first mention of Spinnaker. We haven't had the time to work through all the docs yet, but we're starting to put it on every page going forward.

Reply:
I removed the TM thing from the desc in the front matter and stuck it someplace in the page content.